docs/ops/doc/examples/example_flim_analysis.rst

```diff
@@ -151,17 +142,91 @@ In the panels below, we show script execution with computation restricted to the
         Tau1 = ops.op("transform.hyperSliceView").input(fittedImg, lifetimeAxis, 2).apply()
 
         // Finally, generate a pseudocolored result
-        cMin = getPercentile(Tau1, mask, 5.0)
-        cMax = getPercentile(Tau1, mask, 95.0)
-        pseudocolored = ops.op("flim.pseudocolor").input(lma, cMin, cMax).apply()
+        tLow = getPercentile(Tau1, mask, 5.0)
+        tHigh = getPercentile(Tau1, mask, 95.0)
+        pseudocolored = ops.op("flim.pseudocolor").input(lma, tLow, tHigh, null, null, Pseudocolor.tri2()).apply()
 
         end = System.currentTimeMillis()
         println("Finished fitting in " + (end - start) + " milliseconds")
 
+Adding Calibration and Scale Bars
+---------------------------------
+
+To attach quantitative meaning to the pseudocolored image, we must add calibration and scale bars. We add each within ImageJ using a second SciJava script using the following parameters. Note that ``tLow`` and ``tHigh`` are outputs from the previous script.
+
+With 130 microns across each direction in the dataset, we have 512/130~3.938 pixels per micron.
+
++--------------------------------------+---------------------+
+| Parameter                            | Value               |
++======================================+=====================+
+| tLow                                 |  1.87000572681427   |
```

Comment:
I would also add these as default values for this script. When I first pasted the values in they were rounded to 2, 2 and 4 in the UI🙄 but subsequent runs showed full precision.